### DISCUSSION TITLE
How do i remove AppBar?

### DISCUSSION THREAD

**Discussion by birdcagedout** [flet-dev/flet#959] [Q&A] [answered]
i created and added a AppBar to page and i like to remove the appbar at a certain point.

import flet as ft

def main(page: ft.Page):
	page.appbar = ft.AppBar(
		leading=ft.Icon(ft.icons.ANDROID_OUTLINED, size=80, color=ft.colors.GREEN_600, opacity=0.8),
		automatically_imply_leading=True,
		leading_width=40,
		...

ft.app(target=main, view=ft.FLET_APP_HIDDEN)

i made sure that my app has a list of controls(len=3) but it didnt include appbar i added.
the ways i did tried are as follows:
1. page.clean()
2. page.appbar.open = False; page.appbar.update()
3. page.remove(page.appbar)
all above didnt really made the appbar disappear.

my question is that how do i remove appbar i added to the page?
anyone who knows how please let me know. i appreciate it in advance.

**Comment by birdcagedout** [comment]
i tried several other things and finally i found it. 

page.appbar = None
page.clean()
page.update()

the code above will do.
i'm writing this for the future reference. you're welcome. :)

**Comment by ndonkoHenri** [comment, marked as answer]
```py
page.appbar = None
page.update()
```
will do the job

**Reply by birdcagedout** [reply]
thanks alot.
